setup.cfg

```
@@ -10,15 +10,14 @@ license = MIT
 license_files = LICENSE
 url = https://github.com/con/duct/
 
-# TODO(asmacdo)
-# keywords =
-#     apt
-#     conda
-#     datalad
-#     git-annex
-#     installer
-#     miniconda
-#     neurodebian
+keywords =
+    metrics
+    command-line
+    provenance
+    memory
+    cpu
+    usage
+    time
```

I would sort and add `output-capture`

```suggestion
    command-line
    cpu
    memory
    metrics
    output-capture
    provenance
    time
    usage
```